app/helpers/admin_helper.rb

```diff
@@ -46,4 +46,12 @@ def format_address(*parts)
 
     safe_join(parts.compact_blank, tag.br)
   end
+
+  def identity_transferred_label(identity)
+    if identity.user_id == identity.external_identifier
+      "Original"
+    else
+      "Transferred"
```

Comment:
How do we identify these "Additional" or "Secondary" ones?

Reply:
Ah, wasn't quite what I meant but a really good question! So we can identify an original easy enough.  If the identity was transferred due to de-duping it would most likely have one or more participant_profiles.  The additional identites used to give participants additional email addresses would most likely not have any participant profiles.  But all I was suggesting was that was there something more inclusive/helpful to call the non-original ones rather than "Transferred".  It is only a suggestion, please don't let this hold you up

Reply:
Oh I see. If it's OK, I think I prefer to keep it consistent with the rest of the code for now and review/refactor everything together at a later stage.